app/assets/javascripts/application.js

```diff
@@ -19,26 +19,11 @@
 //= require jquery-ui
 //= require underscore
 //= require gmaps/google
+//= require react
+//= require react_ujs
+//= require components
 //= require_tree .
 //
-// AppKey is from Bing Market Place account
-// var AppKey = ""
-// var encodedAppKey = Base64.encode(':' + AppKey);
-// //
-// $.ajax({
-//   //url: "https://api.datamarket.azure.com/Bing/Search/v1/Web?Query='sushi'&$format=JSON",
-//   url: "https://api.datamarket.azure.com/Bing/Search/v1/News?Query='Vancouver Tech'&$format=JSON",
-//   method: "GET",
-//   headers: {
-//     "Authorization": "Basic " + encodedAppKey
-//   },
-//   error: function() {
-//     alert("Failed bing search");
-//   },
-//   success: function(data){
-//     console.log(data)
-//   }
-// });
 
 var baseUrl = "http://localhost:3000";
```

Comment:
Why is this hardcoded to localhost? You should either use relative urls or fetch this from configurations based on the environment as this will be different in production.